### PR TITLE
Document guarantees of alibuild wrt rebuilds

### PR DESCRIPTION
**Pull request by ktf**

### CODE REVIEW

**Comment by ktf** [comment]
@dberzano, can you see if you agree and then we merge?
@ihrivnac, this should address part of your question by email.

File: docs/user.markdown [hunk 1]
```diff
@@ -248,3 +248,11 @@ or in case you need to be root (_e.g._ on Ubuntu and most Linux distributions
 for convenience):
 
     sudo pip install --upgrade alibuild
+
+In general updating alibuild is safe and it should never trigger a rebuild or
```
Comment (dberzano) [inline docs/user.markdown]: alibuild --> aliBuild

File: docs/user.markdown [hunk 2]
```diff
@@ -248,3 +248,11 @@ or in case you need to be root (_e.g._ on Ubuntu and most Linux distributions
...
+break compilation of older versions of alidist (i.e. we do try to guarantee
+backward compatibility). In no case an update of alibuild will result in the
```
Comment (dberzano) [inline docs/user.markdown]: alibuild --> aliBuild

**Comment by ktf** [comment]
Fixed, thanks. Squashing and merging.

**Pull request merged**